lib/rules/no-ember-super-in-es-classes.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       ) {
         context.report({
           node,
-          message: "Don't use `this._super` in ES classes",
+          message: "Don't use `this._super` in ES classes; instead, you should use super()",
```

Comment:
Perhaps we should just say `super` since the fix could be either of these:

```js
super([arguments]); // calls the parent constructor.
super.functionOnParent([arguments]);
```

Reply:
Thank you. Fixed